src/components/views/dialogs/RoomSettingsDialog.js

```diff
@@ -73,6 +78,15 @@ export default class RoomSettingsDialog extends React.Component {
             "mx_RoomSettingsDialog_rolesIcon",
             <NotificationSettingsTab roomId={this.props.roomId} />,
         ));
+
+        if (shouldShowBridgeIcon) {
+            tabs.push(new Tab(
+                _td("Bridge Info"),
```

Comment:
I wonder if this should just be "Bridges", or "Connected Bridges", or "Integrations"